frontend/src/metabase/core/components/FormField/FormField.styled.tsx

```diff
@@ -24,13 +25,22 @@ export const FieldCaption = styled.div<FormCaptionProps>`
 
 export interface FieldLabelProps {
   hasError: boolean;
+  titleStyle?: "small" | "large";
```

Comment:
Name this style rather than size because it's more than just size. They're completely different styles.

#### Old styles (mostly seen on forms on admin settings with vertical orientation)
<img width="346" alt="image" src="https://user-images.githubusercontent.com/1937582/231551893-0aec390b-dff2-4fbc-b359-a8d07c2bf8c2.png">

#### New style (This matches the [design](https://www.figma.com/file/MPTXmUB6nJ2w0zi4aZTiwC/Prevent-auto-apply-filters-in-slow-dashboards?node-id=1169-4854), and a lot of newer settings)
<img width="343" alt="image" src="https://user-images.githubusercontent.com/1937582/231551744-a98fb55f-85a1-41a5-bea7-3863377fbd06.png">


___

[original comment](https://github.com/metabase/metabase/pull/29569#discussion_r1165260768) from @ranquild 
> Unless there was an explicit design call to change this (1) for this specific place and make this conditional so we have 2 different labels in the app (2) we should not do changes like this.
___
@mazameli can you confirm this is a new style? I've seen this in a number of places e.g. visualization settings.
<img width="637" alt="Screenshot 2023-04-13 at 11 27 16 AM" src="https://user-images.githubusercontent.com/1937582/231731792-f76e2643-7137-45d0-8d13-17665431007b.png">

The only place I think I've seen the old style is in the admin settings forms.